src/gui/RefineCTFPanel.cpp

```diff
@@ -7,7 +7,7 @@ extern MyRefinementResultsPanel *refinement_results_panel;
 
 RefineCTFPanel::RefineCTFPanel( wxWindow* parent )
 :
-RefineCTFParentPanel( parent )
+RefineCTFPanelParent( parent )
```

Comment:
Yes, unfortunately it looks like the "new" approach is somewhat muddled by this particular instance where the wxFormbuilder grandparent is *AssetParentPanel* from which *MyAssetParentPanel* derives, from which My<Name>AssetPanel derives.

Writing this out, I guess the third layer of inheritance doesn't matter, and it could be

<Image,Volume etc>AssetPanel  <-- AssetPanel <-- AssetPanelParent

I guess the real problem is that we're trying to use "Parent" really to mean "From wxFormBuilder." A cleaner approach may just be

SpecificDerivedPanel <-- DerivedPanel <-- PanelFromWX
eg

ImageAssetPanel <-- AssetPanel <-- AssetPanelFromWX

Changing these actually was pretty easy using vscode, so maybe it is worth just going through all of this once while it is fresh in my mind, but I would rather do that in a separate PR.